### PR TITLE
Component Confirm Modal

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
feature https://github.com/ProcessMaker/bpm/issues/458

- Add global component in ProcessMaker.confirmModal(title, message, variant, callback)

test

### CODE REVIEW

**Comment by marcoAntonioNina** [comment]
change 

test

note: 
- the props "message" accept html tags.
- button close not emit values
- buttons cancel and close emit variable "confirm" with values false and true

File: resources/assets/js/designer/components/modalList/modal-forms-list.vue [hunk 1]
```diff
@@ -79,18 +79,21 @@
...
-                        }
-                    )
-                    .then(response => {
-                        this.fetch();
-                    })
+                let that = this;
+                ProcessMaker.confirmModal('Caution!', '<b>Are you sure to delete </b>' + data.title + '?', '', function() {
+                    const CancelToken = ProcessMaker.apiClient.CancelToken;
+                    ProcessMaker.apiClient
+                        .delete('process/' + that.processUid + '/form/' + data.uid,
```
Comment (velkymx) [inline resources/assets/js/designer/components/modalList/modal-forms-list.vue]: This modal needs to be more generic. We will use to any time we delete anything.

Comment (velkymx) [reply resources/assets/js/designer/components/modalList/modal-forms-list.vue]: Not just for deleting processes

**Comment by marcoAntonioNina** [comment]
- variant now affect a buttons
- buttons Cancel and Confirm

image

File: resources/assets/js/designer/components/modalList/modal-forms-list.vue (hunk 1 above)
Comment (marcoAntonioNina) [reply resources/assets/js/designer/components/modalList/modal-forms-list.vue]: the function can be used globally

**Pull request merged**